spec/views/teams/show.html.erb_spec.rb

```diff
@@ -29,6 +29,32 @@
     rendered.should_not have_content(t("helpers.links.assign_ownership"))
   end
 
+  describe 'invite user to team' do
+    describe 'button and modal' do
+      it 'should be rendered for team members' do
```

Comment:
Consider implementing separate tests for button and modal here. Different reasons for button and modal can cause rendering issues. While for the button an authorization problem can lead to issues, for the modal a javascript error can be problematic. (Not that important though.)

Reply:
Good catch, done!